### PR TITLE
📝 Proposed 2022 election schedule.

### PR DESCRIPTION
**Pull request by coderanger**
Draft schedule for review by EOs.

### CODE REVIEW

**Comment by MadhavJivrajani** [comment]
/hold
To prevent any accidental merges, please feel free to remove it anytime!

**Comment by jberkus** [comment]
Three things based on last year's retro:

1. Please add a "nominations due" deadline that's 3 days (or so) before "candidate bios due".
2. The Q&A is for steering + candidates, but not for the community.  It's so that candidates can find out what they're getting into.  Ideally, it's before voting starts so that candidates can still withdraw.  We were confused about this last year, if you recall.
3. We talked about having a community meeting /candidate panel for the whole community *also*, but didn't actually plan one.

**Comment by dims** [comment]
Apart from the question marks in 2 items, the rest of the dates look good.

The steering meets typically on mondays and September 29. 2022 is a Thursday, do we want the steering to call a special meeting or could we align this to the next public steering meeting?

File: elections/steering/2022/README.md
Comment (jberkus) [inline elections/steering/2022/README.md:136]: Add:

Tuesday, August 30 | Candidate Nominations Due.

File: elections/steering/2022/README.md
Comment (jberkus) [inline elections/steering/2022/README.md:135]:
```suggestion
| ???, August ??          | Steering Committee Q+A for the candidates |
```

**Review by jberkus — CHANGES_REQUESTED** [review]

**Comment by coderanger** [comment]
> The steering meets typically on mondays and September 29. 2022 is a Thursday, do we want the steering to call a special meeting or could we align this to the next public steering meeting?

@dims If Monday Oct 3rd would work better for a SC meeting then I think it's fine to bump out the ending dates a bit :) Or we could shorten it, it's not like voting actually takes very long so I'm not super worried about how long voting is open for as long as it's not less than 2 weeks in case people are on vacation.

**Comment by dims** [comment]
@coderanger +1 to early

File: elections/steering/2022/README.md
Comment (liggitt) [inline elections/steering/2022/README.md:135]: this should probably go after the nominations deadline, so the candidates are known, right?

Comment (cblecker) [reply elections/steering/2022/README.md:135]: I think we've previously done this prior to the deadline so that folks can ask questions to see if they actually want to run, and understand more about the job.

Comment (liggitt) [reply elections/steering/2022/README.md:135]: ah, ok... I couldn't remember if it was an open invitation or if candidates were specifically invited

File: elections/steering/2022/README.md
Comment (liggitt) [inline elections/steering/2022/README.md:143]: I'm trying to remember the rationale for trickling the results out in three phases like this.

Unless there are specific actions the non-running steering members and candidates need time to complete in the three days after results are known, it seems simpler and more open to post the results publicly once they are finalized (and any maximal representation issues are resolved?). Would that correspond to Monday, September 26 in this schedule?

That would also remove the need to tightly couple result announcements to a steering meeting (which, as mentioned earlier, is currently scheduled for Monday, October 3rd).

File: elections/steering/2022/README.md
Comment (liggitt) [reply elections/steering/2022/README.md:135]: if the Q+A will be open to potential candidates, keeping this ahead of nomination close seems fine, so feel free to resolve this comment

File: elections/steering/2022/README.md
Comment (jberkus) [reply elections/steering/2022/README.md:143]: Here's the reasons, which may or may not still be valid:

1. The reason for notifying non-running Steering first is in case steering's assistance is needed with any rules hairsplitting, such as maximal representation corner cases, but not limited to that.  The one time I know we needed that notice was Tim's SC vs. CoCC decision.

2. The reason for notifying the candidates was so that there were no outburst-inducing unpleasant surprises when the final results are announced at the steering meeting, particularly considering that it may be 1am for some candidates.

Comment (liggitt) [reply elections/steering/2022/README.md:143]: Thanks for the refresher. I'll defer to the election officers on how much time they think could be needed for 1.

For 2, if results were announced in a meeting, I agree a heads-up to the candidates ahead of time is considerate.

That said, having worn many of the hats that touch this process (community member curious about the results, candidate on pins and needles, non-running member getting an early disclosure), in every case I would have preferred the results be released as soon as they were definite rather than artificially delaying to coordinate with announcing in a steering meeting.

If the election officers think it would simplify the process, and if other reviewers agree, I'd be happy to see this streamlined to something like:

```
| Monday, September 26       | Private announcement of results to SC members not up for election, if needed to resolve maximal representation or committee conflict of interest issues |

| By Wednesday, September 28 | Public announcement of results to candidates, steering committee, and [communication channels]... |
```

If you would rather leave it as-is, that's fine with me as well.

Comment (coderanger) [reply elections/steering/2022/README.md:143]: From having been caught off guard by an election in another community (to be fair it was "surprise you're a candidate" not "surprise you're elected" but still) I think confirming with the candidates privately first is still a good idea. I wouldn't mind if the gap was a few hours rather than a day but because timezones, I don't think we can do that.

**Comment by jberkus** [comment]
@coderanger just so you're prepared, times will still display in Elekto as UTC.

**Comment by coderanger** [comment]
> @coderanger just so you're prepared, times will still display in Elekto as UTC.

Okay, you can interpret "September 29 AoE" as "September 29 11:59:59 @ UTC-12" for computer-y purposes :)

**Comment by coderanger** [comment]
@dims @kaslin Last call on thoughts before we make this official :)

**Comment by dims** [comment]
@coderanger Let's 🚢 🚢 🚢 🚢  it!

**Comment by kaslin** [comment]
+1 to what @dims said!

**Pull request merged**
